Dockerfile.deployment

```diff
@@ -65,7 +64,7 @@ ARG GIT_SHA
 RUN printf "%s" "$GIT_SHA" > version
 
 RUN useradd -m osuweb
-RUN chown -R osuweb bootstrap/cache storage /var/lib/nginx
```

Comment:
`/var/lib/nginx` still has wrong permission

Reply:
what purpose does chowning it to `osuweb` serves?

Reply:
i focused on octane (i do not run nginx assets at all) so i may not have caught everything re: nginx and did not see any obvious sign of nginx needing to write to /var/lib/nginx